internal/observability.go

```diff
@@ -28,80 +28,86 @@ import (
 	"go.opencensus.io/stats/view"
 	"go.opencensus.io/tag"
 	"go.opencensus.io/trace"
-
-	commonpb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
-	tracepb "github.com/census-instrumentation/opencensus-proto/gen-go/trace/v1"
 )
 
 var (
-	tagKeyReceiverName, _ = tag.NewKey("opencensus_receiver")
-	tagKeyExporterName, _ = tag.NewKey("opencensus_exporter")
-)
-
-var mReceivedSpans = stats.Int64("oc.io/receiver/received_spans", "Counts the number of spans received by the receiver", "1")
+	tagKeyTraceReceiverName, _ = tag.NewKey("opencensus_trace_receiver")
```

Comment:
nit: we already have "oc" on the metric name, and probably on namespace too, perhaps just "receiver", since we counting spans it can't apply to metrics.

Reply:
Rename it to oc_receiver. We will encode the signal (trace/metrics/etc) in the value, for example "oc_trace", reason is that if we extract metrics from Spans we want to know what was the source of the exported metrics, and it was not "oc" because that may mean the metrics came from "oc_metrics".